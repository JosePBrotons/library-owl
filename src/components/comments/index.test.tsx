import React from 'react';
import renderer from 'react-test-renderer';
import Comments from '.';

test('renders comments component correctly', () => {
    const commentsCmp = renderer.create(<Comments />).toJSON();
    expect(commentsCmp).toMatchSnapshot();
});
