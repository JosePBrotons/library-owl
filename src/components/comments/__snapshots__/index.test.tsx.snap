// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders comments component correctly 1`] = `
Array [
  <View
    style={
      Object {
        "backgroundColor": "rgba(235, 246, 251, 1)",
        "height": 110,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/img/navbar.png",
        }
      }
      style={
        Object {
          "height": 111,
          "position": "absolute",
          "top": 0,
          "width": "100%",
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "marginVertical": 33,
          "paddingHorizontal": 20,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "rgba(255, 255, 255, 1)",
                "fontSize": 20,
              },
              undefined,
              Object {
                "fontFamily": "FontAwesome",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          ÔÅì
        </Text>
      </View>
      <Text
        style={
          Object {
            "color": "rgba(255, 255, 255, 1)",
            "fontFamily": "Nunito-Bold",
            "fontSize": 22,
            "lineHeight": 30,
            "textAlign": "center",
          }
        }
      >
        [MISSING "EN.GLOBAL.COMMENTS" TRANSLATION]
      </Text>
      <View
        style={
          Object {
            "height": 20,
            "width": 20,
          }
        }
      />
    </View>
  </View>,
  <RCTScrollView
    data={Array []}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    showsVerticalScrollIndicator={false}
    stickyHeaderIndices={Array []}
    style={
      Object {
        "backgroundColor": "rgba(235, 246, 251, 1)",
        "flex": 1,
        "flexDirection": "column",
        "paddingHorizontal": 20,
      }
    }
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>,
]
`;
